x-pack/plugins/search_inference_endpoints/common/doc_links.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DocLinks } from '@kbn/doc-links';
+
+class InferenceEndpointsDocLinks {
+  public nlpImportModel: string = '';
+  public supportedNlpModels: string = '';
+
+  constructor() {}
+
+  setDocLinks(newDocLinks: DocLinks) {
+    this.nlpImportModel = newDocLinks.ml.nlpImportModel;
+    this.supportedNlpModels = newDocLinks.ml.supportedNlpModels;
```

Comment:
@szabosteve owns the ML docs, will be better placed to answer this one

Reply:
Those two docs pages (model import and third-party model support) do not exist in Serverless docs.

The reason is the Docs team doesn't want to duplicate content in the Serverless docs that is already published in the Stateful docs because of maintainability issues. It'll be easier to provide the same set of content for Serverless and Stateful (if relevant) after the MDX migration is done (later this year).

As in this case there are no Serverless docs to link to, I think using the Stateful docs links is an acceptable solution.